lq1/default.cfg

```diff
@@ -58,13 +59,44 @@ bind	"-"		"sizedown"
 +speed
 
 //Cvars
-gl_texturemode "GL_NEAREST_MIPMAP_LINEAR"
-gl_texture_anisotropy "8"
-registered "1""
-vid_fsaamode "1"
-vid_fsaa "4"
-volume "0.7""
-
+gl_texturemode gl_nearest_mipmap_linear
+gl_texturemode_sky gl_nearest
+gl_texturemode_hud gl_nearest
+gl_texturemode2d gl_nearest
+gl_overbright 1
+gl_overbright_model 1
+gl_smoothfont 0
+gl_texture_anisotropy 16
+vid_width "1280"
+vid_height "800
+vid_fsaamode 0
+vid_fsaa 8
+vid_conautoscale 3
+scr_pixelaspect 1
+scr_crosshairscale 3
+scr_menuscale 3
+scr_conscale 3
+scr_sbarscale 3
+cl_stainmaps 0
+cl_decals 0
+cl_alwaysrun 0
+cl_particles_quake 1
+cl_beams_polygons 0
+r_font_linear 0
+r_font_postprocess_mono 1

```

Comment:
What does this do?

Reply:
not fully sure myself but spike recomended it ![SmartSelect_20240306_011436_Discord.jpg](https://github.com/MissLavender-LQ/LibreQuake/assets/51702517/4c962a51-7320-46df-99b6-0338b94f8582)

Reply:
This message specifically states that it creates problems with scaling modes, though

Reply:
this was in response to my ask of how to disable ftes ugly font filtering 
as soon as the user sets their res anything 800x600 or above this is not an issue